static/bidder-params/eplanning.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "EPlanning Adapter Params",
+  "description": "A schema which validates params accepted by the EPlanning adapter",
+  "type": "object",
+  "properties": {
+    "ssp_space_id": {
+      "type": ["integer", "string"],
```

Comment:
This doesn't line up with the code.

In `imp_eplanning.go`, you've got:

```
SspSpaceId int `json:"ssp_space_id"`
```

So... if someone sends a `string` here, the `Unmarshal` will fail.